install_config/provisioners.adoc

```diff
@@ -0,0 +1,198 @@
+[[install-config-provisioners]]
+= Deploying External Persistent Volume Provisioners
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+An external provisioner is an application that enables dynamic provisioning for
+a particular storage provider. External provisioners can run alongside the
+provisioner plug-ins provided by {product-title} and are configured in a
+similar way using xref:../install_config/persistent_storage/dynamically_provisioning_pvs.adoc[_StorageClasses_]. However, being external, they can be deployed
+and updated independently of {product-title} on an as-needed basis.
+
+[[provisioners-before-you-begin]]
+== Before You Begin
+
+An Ansible playbook is available to deploy and upgrade external provisioners. You
+should familiarize yourself with the
+xref:../install_config/install/advanced_install.adoc#install-config-install-advanced-install[Advanced Installation]
+section. This provides information for preparing to use Ansible and includes
+information about configuration. Parameters are added to the Ansible inventory
+file to configure which external provisioners to install and various areas of
+them.
+
+[[provisioners-ansible-role]]
+== External Provisioners Ansible Role
+
+The OpenShift Ansible `openshift_provisioners` role configures and deploys
+external provisioners using the variables from the
+xref:../install_config/install/advanced_install.adoc#configuring-ansible[Configuring Ansible]
+inventory file. You must specify which provisioners to install by overriding
+their respective "install" variables to `true`.
+
+[[provisioners-ansible-variables]]
+=== External Provisioners Ansible Variables
+
+The following is a list of role variables that apply to all provisioners for
+which the "install" variable is `true`.
+
+.Ansible Variables
+[options="header"]
+|===
+
+|Variable |Description
+
+|`openshift_provisioners_install_provisioners`
+|Deploy provisioners that have their respective "install" variables `true`, if
+`true`. Otherwise, undeploy them.
+
+|`openshift_provisioners_image_prefix`
```

Comment:
the thinking is that, everyone will want to consider installing registry/metrics, etc. regardless of what platform they are deploying on. But only the few people who are using AWS and also want to use EFS as storage will care to install the provisioner. So I didn't want to clutter the install instructions for people who it is not relevant to.

I don't feel that strongly about it though, let's hook it up to the installation next release to give it more visibility